### PR TITLE
Update Dragoncurse Castle + make "Fews" consistent

### PR DESCRIPTION
**Pull request by qwrtln**

### CODE REVIEW

File: clash/dragoncurse_castle.tex
Comment (cesarpim) [inline clash/dragoncurse_castle.tex:1]: Hi, I'm profmamadu (author of Dragoncurse Castle scenario). Please correct that obelisks don't give 2 building materials, they give 2 resource dice results (i.e. roll 2 resource dice and get both results). They are supposed to be a high incentive location. The symbol I used in my original text was not the building materials symbol, it was the resource dice symbol.
Thanks!

Comment (qwrtln) [reply clash/dragoncurse_castle.tex:1]: Will do, thanks for reporting.

Comment (qwrtln) [reply clash/dragoncurse_castle.tex:1]: Done.

Comment (cesarpim) [reply clash/dragoncurse_castle.tex:1]: thank you!

**Review by tyson-heckert — APPROVED** [review]
looks great!

**Pull request merged**